app/proxy/handlers.go

```diff
@@ -40,18 +40,21 @@ func (rh *RequestHandler) Handle(w http.ResponseWriter, r *http.Request) {
 
 	var walletID string
 	if config.AccountsEnabled() {
-		retriever := users.NewWalletService()
-		auth := users.NewAuthenticator(retriever)
-		walletID, err = auth.GetWalletID(r)
+		q, err := NewQuery(body)
```

Comment:
I thought all accounts logic is no longer in use?

Reply:
What accounts logic?

Reply:
I think  maybe its just a naming issue, but I was referring to the `if config.AccountsEnabled()`.

Reply:
Right, that was a naming discrepancy. I just removed the config flag altogether since there's no reason to support both logical pathways anymore.